README.md

```diff
@@ -387,6 +391,79 @@ $ make flash
 [here](https://github.com/96boards/documentation/wiki/HiKeyUEFI#flash-binaries-to-emmc-))
 
 The board is ready to be booted.
+#### 5.5.2 Debian based / 96boards RPB (experimental)
+Start by getting the source and toolchain (see above), then continue by
+downloading the system image (root fs). There are two versions, the one that
+works best has a click through license which means that you have to manually
+download the system image from Linaro release pages and then gunzip and rename it
+so it ends up in the expected directory with the correct name. Then you just run
```

Comment:
The make command will give you the exact name and location:
https://github.com/jbech-linaro/build/blob/debian_fs/hikey_debian.mk#L269-L270